apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -71,6 +72,41 @@ const CurriculumCatalog = ({
     setExpandedCardKey(expandedCardKey === key ? null : key);
   };
 
+  // Get the top recommended similar curriculum based on the curriculum with the given
+  // curriculumKey
+  const getRecommendedSimilarCurriculum = curriculumKey => {
```

Comment:
I'm defining the `getRecommendedSimilarCurriculum` function in `CurriculumCatalog.jsx` and passing it down to `ExpandedCurriculumCatalogCard.jsx` so that later I can more easily implement a way to cache recommender results.